packaging_automation/dbconfig.py

```diff
@@ -4,7 +4,7 @@
 from attr import dataclass
 from sqlalchemy import Column, INTEGER, TIMESTAMP, TEXT
 from sqlalchemy import create_engine
-from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import declarative_base
```

Comment:
The import order here looks weird. You can actually use isort to sort the imports properly.

For the time being, it is ok. However it would be better to introduce isort and flake8 support in a separate PR to improve the code readability a little bit. You can check out https://github.com/citusdata/citus/pull/6700 that sets a great example for a similar goal.

Reply:
Thanks I will use isort in the further PR's. There is already prospector which includes flake8